scripts/check_lint.sh

```diff
@@ -10,10 +10,13 @@ set -e
 echo "Running $0"
 
 DOCKER_CMD=${DOCKER_CMD:-docker}
+GOLANGCI_LINT_IMAGE="golangci/golangci-lint:v1.23.1"
```

Comment:
1.27.0 is available (I can't recall the exact version, but I think 1.23.1 doesn't support Go 1.14)

Reply:
done. updated to 1.27.0